### PR TITLE
correct tf.summary.scalar illegal name `learning rate`

### PR DESCRIPTION
**Pull request by robrao**
Corrected issue listed below.

`INFO:tensorflow:Summary name learning rate is illegal; using learning_rate instead.`

### CODE REVIEW

**Comment by nealwu** [comment]
Thanks!
